### PR TITLE
Creating the log directory on install

### PR DESCRIPTION
**Pull request by adamb0mb**
I was getting this error on zk startup:
```
[adamp@devzk02 ~]$ sudo service zookeeper start
Starting zookeeper Server (zookeeper):                     [  OK  ]
log4j:ERROR setFile(null,true) call failed.
java.io.FileNotFoundException: /var/log/zookeeper/zookeeper.log (No such file or directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:142)
	at org.apache.log4j.FileAppender.setFile(FileAppender.java:294)
	at org.apache.log4j.RollingFileAppender.setFile(RollingFileAppender.java:207)
	at org.apache.log4j.FileAppender.activateOptions(FileAppender.java:165)
	at org.apache.log4j.config.PropertySetter.activate(PropertySetter.java:307)
	at org.apache.log4j.config.PropertySetter.setProperties(PropertySetter.java:172)
	at org.apache.log4j.config.PropertySetter.setProperties(PropertySetter.java:104)
	at org.apache.log4j.PropertyConfigurator.parseAppender(PropertyConfigurator.java:809)
	at org.apache.log4j.PropertyConfigurator.parseCategory(PropertyConfigurator.java:735)
	at org.apache.log4j.PropertyConfigurator.configureRootCategory(PropertyConfigurator.java:615)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:502)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:547)
	at org.apache.log4j.helpers.OptionConverter.selectAndConfigure(OptionConverter.java:483)
	at org.apache.log4j.LogManager.<clinit>(LogManager.java:127)
	at org.slf4j.impl.Log4jLoggerFactory.getLogger(Log4jLoggerFactory.java:73)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:242)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:254)
	at org.apache.zookeeper.server.quorum.QuorumPeerMain.<clinit>(QuorumPeerMain.java:64)
```

This simply just creates that directory (in the model that other directories are being created)

### CODE REVIEW

File: templates/default/environment-defaults.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@ ZOOCFGDIR="$ZOODIR/conf"
 CLASSPATH="$ZOODIR/build/classes:$ZOODIR/bin/build/lib/*.jar:$ZOODIR/lib/slf4j-log4j12-1.6.1.jar:$ZOODIR/lib/slf4j-api-1.6.1.jar:$ZOODIR/lib/netty-3.7.0.Final.jar:$ZOODIR/lib/log4j-1.2.16.jar:$ZOODIR/lib/jline-0.9.94.jar:$ZOODIR/zookeeper-3.4.6.jar:$ZOODIR/src/java/lib/*.jar:$ZOOCFGDIR"
 
 ZOOCFG="$ZOOCFGDIR/zoo.cfg"
-ZOO_LOG_DIR="/var/log/$NAME"
+ZOO_LOG_DIR="<%= node[:zookeeper][:log_dir] %>"
```
Comment (jeffbyrnes) [inline templates/default/environment-defaults.erb]: Especially fond of this change. Are there any other service scripts that could take advantage of this?

Comment (adamb0mb) [inline templates/default/environment-defaults.erb]: This covered it. I thought maybe some of the changes from #148 might have added a new one, but it appears not. The service files get it from the environment variable `$ZOO_LOG_DIR` set in `environment-defaults.erb` file.

**Comment by jeffbyrnes** [comment]
@adamb0mb looks like this needs a rebase; would you mind? I’d like to cut a new version w/ the CentOS support etc. sometime today.

**Comment by adamb0mb** [comment]
Done. Cut away!

**Comment by jeffbyrnes** [comment]
@adamb0mb hmm, looks like the `templates/default/zookeeper.initd.erb` template snuck in there (unneeded, we have `templates/default/zookeeper.sysv.erb`). Would you mind deleting that file? Then we should be good!

**Comment by adamb0mb** [comment]
Good catch! Thank you.

**Pull request merged**